### DISCUSSION TITLE
Custom snap areas

### DISCUSSION THREAD

**Discussion by genebean** [rxhanson/Rectangle#291] [Ideas]
Any chance of adding an option to define custom snap areas similar to what Better Snap Tool has?

**Comment by rxhanson** [comment]
I would like to add this, but currently don’t have the bandwidth to do it. Pull requests welcome

**Comment by neveskiet** [comment]
I too would really like the ability to have custom snap areas. Or at least pre-defined snap areas, like Microsofts Power Toys in Windows.

**Comment by gcdbox** [comment]
This would be awesome, currently I'd like to have snap areas for sixth windows' sizes
